### PR TITLE
Add a checklist to issues template

### PR DESCRIPTION
**Pull request by bhavin192**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/propose-a-talk.md [hunk 1]
```diff
@@ -12,16 +12,24 @@ assignees: ''
...
 <!-- Please add estimated duration of the talk including Q&A -->
 
-**Prerequisites**
+#### Prerequisites
 <!-- If you have any specific requirement from audience -->
 <!-- for the talk i.e. software, tools set up on machine etc, please -->
 <!-- include it here -->
+
+#### Checklist
+[Place an '[x]' (no spaces) in all the fields you agree to. You can keep the fields blank on which you don't agree.]
```
Comment (akshay196) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: This line should be comment.

File: .github/ISSUE_TEMPLATE/propose-a-talk.md [hunk 2]
```diff
@@ -12,16 +12,24 @@ assignees: ''
...
+
+The talk/workshop speaker agrees to,
+- [ ] Share the slides, code snippets and other material used during the talk
+- [ ] If the talk is recorded, you grant the permission to release the video on [PythonPune's YouTube channel](https://www.youtube.com/channel/UCWjk7oGWV9eknuOzC20dyiQ) under [CC-BY-4.0 license](https://creativecommons.org/licenses/by/4.0/)
+- [ ] Not do any hirig pitches during the talk and follow the Code of Conduct
```
Comment (akshay196) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: Typo. It should be `hiring`.

Comment (akshay196) [inline .github/ISSUE_TEMPLATE/propose-a-talk.md]: How about hyper-link Code of Conduct?

Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: Yep, I forgot to add it

File: .github/ISSUE_TEMPLATE/propose-a-talk.md (hunk 1 above)
Comment (bhavin192) [reply .github/ISSUE_TEMPLATE/propose-a-talk.md]: This was taken from, https://github.com/helm/charts/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Let's make it a comment

**Pull request merged**